docs/current_docs/ci/integrations/apple-container.mdx

```diff
@@ -0,0 +1,53 @@
+---
+slug: /ci/integrations/apple-container
+---
+
+# Apple's Container Project
+
+Dagger can be used with any OCI-compatible container runtime, including [`container` from Apple](https://github.com/apple/container).
+
+:::warning
+The support for Apple's `container` is still early and experimental.
+:::
+
+## How it works
+
+By default, Dagger tries to invoke the `docker` executable at some points during the invocation of `dagger`. Since `container` is partially CLI-compatible with Docker, some functionality works out of the box, and there is a way to get running by combining a symbolic link named `docker` pointing to the `container` executable, together with the approach of [Custom Runner support](https://docs.dagger.io/configuration/custom-runner).
+
+## Prerequisites
+
+- `container` installed and API server running on the host system, through `container system start`.
+
+## Example
+
+To use `container` with Dagger:
+
+- create a symbolic link in a superior `PATH` position as below:
+
+    ```shell
+    sudo ln -s $(which container) /usr/local/bin/docker
+    ```
```

Comment:
Hey @jpadams had an idea - would having a `apple-container://` prefix for `_EXPERIMENTAL_DAGGER_RUNNER_HOST` be useful here?

If so, happy to get that setup, it should be pretty simple.